dat/missions/neutral/pirbounty_dead.lua

```diff
@@ -200,8 +205,10 @@ function land ()
    if job_done and planet.cur():faction() == paying_faction then
       local pay_text
       if target_killed then
+         misn.appendLog("Payment of " .. math.floor(credits) .. " received for death of pirate")
          pay_text = pay_kill_text[ rnd.rnd( 1, #pay_kill_text ) ]
       else
+         misn.appendLog("Payment of " .. math.floor(credits) .. " received for capture of pirate")
```

Comment:
Assuming the log is supposed to be a useful reference for important past events, this is just cluttering it up with unnecessary information.